code-challenges.test.js

```diff
@@ -52,5 +130,15 @@ const numbersToAdd2 = [0, 7, -8, 12]
 const numbersToAdd3 = []
 // Expected output: []
 
+// create a variable 
+// use the .map method to iterate through an array taking in the last value and adding the next for the length of the array
+
+
+function sumOfAll (array) {
```

Comment:
Great approach. I bet you can do this with one return statement without  a variable.